module-2/backend/week_four.md

```diff
@@ -10,14 +10,29 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+A small piece of data sent to a user and stored on their client as to save any type of information about their 'session' as HTTP is stateless
 * What’s the difference between a session and a cookie?
+A session is stored on the server whereas a cookie is stored in the browser. This makes sessions much more (but still not totally) secure than cookies.
```

Comment:
Sessions are also stored client side, but they are encrypted. You can set a session to save server side (see [this article](https://www.justinweiss.com/articles/how-rails-sessions-work/) for more info), but I believe the default behavior in Rails is to store it in a cookie.